lib/Source.js

```diff
@@ -6,9 +6,12 @@ const { toTerm } = require('./utils')
 const cubesQuery = require('./query/cubes.js')
 const viewListQuery = require('./query/views.js')
 const View = require('./View')
+const createPrefixedSparqlClient = require('./PrefixedSparqlClient.js')
+
+const DEFAULT_QUERY_PREFIX = '#pragma describe.strategy cbd'
 
 class Source extends Node {
-  constructor ({ parent, term, dataset, graph, endpointUrl, sourceGraph, user, password, queryOperation }) {
+  constructor ({ parent, term, dataset, graph, endpointUrl, sourceGraph, user, password, queryOperation, queryPrefix }) {
```

Comment:
A useful syntactic sugar for argument destructuring is to set default
```suggestion
  constructor ({ parent, term, dataset, graph, endpointUrl, sourceGraph, user, password, queryOperation, queryPrefix = DEFAULT_QUERY_PREFIX }) {
```